cf/test/test_cfa.py

```diff
@@ -28,6 +28,7 @@ def setUp(self):
             | stat.S_IWUSR,  # writing
         )
 
+    @unittest.skipIf(True, "wait until TODOCFAVER")
```

Comment:
```suggestion
```

since we're all done in that respect now (I checked it passes without the skip locally, to be sure).